stripe/src/main/java/com/stripe/android/model/CountryCode.kt

```diff
@@ -0,0 +1,25 @@
+package com.stripe.android.model
+
+import android.os.Parcelable
+import androidx.annotation.Size
+import kotlinx.parcelize.Parcelize
+import java.util.Locale
+
+@Parcelize
+data class CountryCode(
+    @Size(2) private val letters: String,
+    val twoLetters: String = letters.toUpperCase(Locale.getDefault())
+) : Parcelable {
```

Comment:
I would pull out `twoLetters` into a property with backing value

e.g.

```
@Parcelize
data class CountryCode(
    @Size(2) private val letters: String
) {
    val twoLetters: String get() = letters.toUpperCase(Locale.getDefault())
}
```

Reply:
I would also rename `letters` to `value` to make it clear that it's the source of truth.